.github/workflows/test.yml

```
@@ -185,3 +185,25 @@ jobs:
       - name: Test NetworkX
         run: |
           pytest -n auto --doctest-modules --durations=10 --pyargs networkx
+
+  slow-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+      - name: Set up Python
+        uses: actions/setup-python@v6
+        with:
+          python-version: "3.13"
+
+      - name: Install packages
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install .[default,test]
+          python -m pip list
```

Nit: I believe we can do as the `prerelease` job does and remove the `python -m`. The other jobs keep it. Whatever you think looks best works for me!
```suggestion
          pip install --upgrade pip
          pip install .[default,test]
          pip list
```